tests/integration/TestCase.php

```diff
@@ -566,9 +566,13 @@ public function copyFiles($files)
                     $this->files->makeDirectory($directory, 0755, true);
                 }
 
+                if ($this->files->exists($destination)) {
+                    $this->files->delete($destination);
+                }
+
                 $this->files->copy($source, $destination);
 
-                usleep(1000 * 100); // 100ms
+                /* usleep(1000 * 100); // 100ms */
```

Comment:
@ifox here

Reply:
😱

Reply:
I honestly was wondering why the tests took so long before. Guessing it had something to do with testbench.